### DISCUSSION TITLE
Portuguese Translation

### DISCUSSION THREAD

**Discussion by OsodracPT** [sovenor/bitcoin-rocks#2] [Ideas]
I'm proposing adding Portuguese 🇵🇹 translation support for the project to increase accessibility and usability for Portuguese-speaking users.

### Benefits:

Enhanced Accessibility: Portuguese-speaking users can better understand and contribute to the project.
Global Outreach: Facilitates outreach to Portuguese-speaking communities, potentially expanding the project's user base and contributors.

**Comment by sovenor** [comment]
Portuguese would be a great addition, especially since we already have [Portuguese bitcoin sticker files](https://bitcoin.rocks/sticker-files/portuguese/) available.

Based on discussion #3 the [translation guide](https://github.com/sovenor/bitcoin-rocks/blob/main/CONTRIBUTING.md) is likely to change. However, any translations that are completed in the current format (by editing the HTML pages) will be manually moved to the new system by me.

Would you like to help with some of the Portuguese translations?

**Reply by OsodracPT** [reply]
Working on it

**Reply by sovenor** [reply]
Fantastic, thank you! Please submit a pull request when the minimum required files have been translated.

**Reply by sovenor** [reply]
Hi, please pause all translation work until translations are more scalable as noted in discussion #3 

I thought this would be an easy migration in the future but it turns out it will not be. Sorry for the inconvenience.

**Reply by sovenor** [reply]
Hi @OsodracPT if you'd like to get started on this again, please check out our new [Translations guide](https://github.com/sovenor/bitcoin-rocks/blob/main/CONTRIBUTING.md) and [make a new discussion titled "Portuguese" here.](https://github.com/sovenor/bitcoin-rocks/discussions/categories/translations)

**Comment by sovenor** [comment]
Closing this discussion now that we have a specific Translations forum for this. Those following along can move to the [new Portuguese discussion.](https://github.com/sovenor/bitcoin-rocks/discussions/13)
